pyairtable/api/base.py

```diff
@@ -15,18 +15,27 @@ class Base(ApiAbstract):
 
     base_id: str
 
-    def __init__(self, api_key: str, base_id: str, timeout=None):
+    def __init__(
+        self,
+        api_key: str,
+        base_id: str,
+        timeout=None,
+        max_records_per_request=DEFAULT_MAX_RECORDS_PER_REQUEST,
+    ):
         """
         Args:
             api_key: |arg_api_key|
             base_id: |arg_base_id|
 
         Keyword Args:
             timeout(``Tuple``): |arg_timeout|
+            max_records_per_request: |arg_max_records_per_request|
         """
 
         self.base_id = base_id
-        super().__init__(api_key, timeout=timeout)
+        super().__init__(
+            api_key, max_records_per_request=max_records_per_request, timeout=timeout
+        )
```

Comment:
Carrying through my suggestions for keyword-only arguments...
```suggestion
        super().__init__(api_key, **kwargs)
```